src/image/loading_displaying.js

```diff
@@ -25,6 +25,9 @@ import '../core/error_helpers';
  * that links in your sketch. Loading an image from a URL or other
  * remote location may be blocked due to your browser's built-in
  * security.
+ * <br><br>
```

Comment:
you can simply leave this line empty and the empty line after the paragraph break will be added